### PR TITLE
Cknieling/package path crash

### PR DESCRIPTION
**Pull request by chr11115**
- Fix crash that could happen when using an invalid file path
- Added Error if rulefile does not exist
- Add Errors from updating the rulefile to the problem view

### CODE REVIEW

**Review by mistafunk — CHANGES_REQUESTED** [review]
bug repro:
1. assign RPK to shape
1. break it by e.g. removing the last "k" character -> newly implemented error message appears correctly and model vanishes
1.  again add the "k" in the field (without using the browse button)
1. No refresh is triggered (manually clicking on the reload button works though)

File: src/serlio/modifiers/PRTModifierAction.cpp [hunk 1]
```diff
@@ -281,15 +281,24 @@ short getDefaultEnumIdx(const prt::Annotation* annot, const PRTEnumDefaultValue&
...
 				MGlobal::displayWarning(error.errorString.c_str());
+			}
 		}
 	}
 	return false;
 }
 
+CGACErrors createCGACErrorFromString(const MString& stringMessage) {
+	CGACErrors cgacErrors;
+	const auto [it, wasInserted] = cgacErrors.try_emplace({prt::CGAErrorLevel::CGAERROR, true, stringMessage.asWChar()}, 1);
```
[the remark this answers is not in the document]
Comment (chr11115) [reply src/serlio/modifiers/PRTModifierAction.cpp]: fixed

File: src/serlio/modifiers/PRTModifierAction.cpp [hunk 2]
```diff
@@ -657,28 +672,43 @@ MStatus PRTModifierAction::updateRuleFiles(const MObject& node, const MString& r
 	mRuleAttributes.clear();
 	PRTContext::get().mPRTCache.get()->flushAll();
 
+	if (!std::filesystem::exists(mRulePkg.asWChar())) {
```
[the remark this answers is not in the document]
Comment (chr11115) [reply src/serlio/modifiers/PRTModifierAction.cpp]: fixed

**Comment by chr11115** [comment]
Fixed the automatic reload when the path variable is changed back after changing it from a broken state

**Review by mistafunk — APPROVED** [review]
LGTM, feel free to merge.

**Pull request merged**